easybuild/easyconfigs/r/rclone/rclone-1.56.0.eb

```diff
@@ -0,0 +1,37 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'MakeCp'
+
+name = 'rclone'
+version = '1.56.0'
+#versionsuffix = '-amd64'
```

Comment:
Please keep the version suffix for this easyconfig as the binaries are specific to `amd64`.